styles/mobile_phones.css

```diff
@@ -23,6 +23,9 @@ nav ul {
     padding-bottom: 1em;
     align-items: center;
 }
+nav a {
+    border: none;
+}

```

Comment:
You need to undo the main (although transparent) border too in order to prevent the nav elements from popping up and down when you hover